### PR TITLE
Styleguide | Add auto focus for modal

### PR DESCRIPTION
**Pull request by mmlumba**
This fix prevents the user from accessing other DOM elements on the page when a modal is activated.
Connects #680

### CODE REVIEW

File: client/app/components/Modal.jsx [hunk 1]
```diff
@@ -37,12 +37,26 @@ export default class Modal extends React.Component {
     }
   }
 
+  getModalId = () => {
```
Comment (amprokop) [inline client/app/components/Modal.jsx]: Is this an unrelated change to the others in here? It's not mentioned in the PR, is it? Not sure why this is needed.

Comment (mmlumba) [reply client/app/components/Modal.jsx]: Oh this was used for my first attempt at the focus (focusing on the div when the modal opens instead of the close button) but forgot to remove. I'll remove it then merge
